src/test/java/HomeWork4/tests/CheckoutInfoTests.java

```diff
@@ -0,0 +1,19 @@
+package HomeWork4.tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CheckoutInfoTests extends BaseTest {
+    @Test
+    public void CheckoutInfoTest ()  {
+        String productName = "Sauce Labs Onesie";
+        String productName2 = "Sauce Labs Backpack";
```

Comment:
Please avoid using numbers in variable / method names, use literals instead: firstProductName, secondProductName etc.